internal/storage/disk/index/index.go

```diff
@@ -36,9 +40,11 @@ type Index interface {
 	GetFiles() []string
 	GetAllCompilationUnits(context.Context) <-chan *policy.CompilationUnit
 	Clear() error
+	GetPolicies(context.Context, storage.PolicyFilter) ([]*policy.Wrapper, error)
 }
 
 type index struct {
+	dir          string
```

Comment:
It seems like this addition is against the abstraction here and I'm not particularly happy breaking it here. But in order to get file creation date, we require the absolute path of the file. As stdlib does not provide a method to access file creation date out of the box.

Reply:
Discussed offline, the creation date will be deferred to another PR.